server/aws-lsp-codewhisperer/src/language-server/chat/chatController.ts

```diff
@@ -330,6 +339,18 @@ export class ChatController implements ChatHandlers {
         return chatEventParser.getChatResult()
     }
 
+    updateConfiguration = async () => {
+        try {
+            const qCustomizationConfig = await this.#features.lsp.workspace.getConfiguration(Q_CONFIGURATION_SECTION)
```

Comment:
The Typescript type for expected configuration object defined in https://github.com/aws/language-servers/pull/462#discussion_r1777211491 can be used in calls to read documentation like here.